src/styles/theme.ts

```diff
@@ -131,8 +131,8 @@ const theme = deepMerge(generate(16), {
         light: '#375b70',
       },
       'background-contrast': {
-        dark: '#15253111',
-        light: '#15253111',
+        dark: '#15253150',
+        light: '#15253150',
```

Comment:
Assuming that's the main container background color, can we please use [darkBlueDarker6 (#10212c)](https://github.com/giantswarm/happa/blob/master/src/styles/theme.ts#L26)?

Reply:
BTW, I didn't intend to make all accordions use that background color. Just the one used for this purpose.

Reply:
Updated screenshots.